### PR TITLE
Added angular & framer pages and updated schedule

### PR DESCRIPTION
**Pull request by nicoleang**

### CODE REVIEW

File: angular/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: guide
+title: "Frontend Development with Angular.js"
+subject: angular
+---
+
+# Frontend Development <br>with Angular.js
+
+
+AngularJS is a powerful tool to create web-applications that are dynamic and efficient. Many of the most popular websites on the web use it, including its creator, Google, and Lyft.  This framework extends the capabilities of HTML code, while making data binding easier. AngularJS will change the way you think about front-end development.
```
Comment (jez) [inline angular/index.md]: Can you hard wrap your lines to 80 characters? (i.e., insert a newline if the line length exceeds 80 characters)

There's usually a way to get your editor to do this. You might want to see this stack overflow answer for how to do it in Sublime:

http://stackoverflow.com/questions/12466430/what-is-the-usual-way-to-reformat-paragraph-in-sublime-text-2/17159320#17159320

In vim, you can use `V}gq`.

File: index.md [hunk 2]
```diff
@@ -40,14 +40,26 @@ free__, so that anyone can see it.
...
-| [Frontend Engineering with JavaScript][frontend]  | n/a | n/a  | 3:00 p.m. - 4:30 p.m. |
-| *break for dinner*                                | n/a | n/a  | 4:30 p.m. - 5:00 p.m. |
-| [Backend Engineering with Flask][backend]         | n/a | n/a  | 5:00 p.m. - 6:30 p.m. |
-| [Making Your Site Public: Deployment][deployment] | n/a | n/a  | 1:30 p.m. - 3:00 p.m. |
-| [Crafting Well-Designed Sites][design]            | n/a | n/a  | 3:00 p.m. - 4:30 p.m. |
+| [HTML and CSS][html]                              | Sun | 11/6 | 1:30 p.m. - 3:00 p.m. |
+| [Frontend Engineering with JavaScript][frontend]  | Sun | 11/6 | 3:00 p.m. - 4:30 p.m. |
+| [Crafting Well-Designed Sites][design]            | Sun | 11/6 | 4:30 p.m. - 6:00 p.m. |
+| *break for dinner*                                | Sun | 11/6 | 6:00 p.m. - 6:30 p.m. |
+| [Backend Engineering with Flask][backend]         | Sun | 11/6 | 6:30 p.m. - 8:00 p.m. |
```
Comment (jez) [inline index.md]: So I just noticed that since we have two tables (one for Saturday, one for Sunday), we can probably get rid of the "Day" column.

**Review by jez — CHANGES_REQUESTED** [review]
I left some notes inline.

Overall it looks great!

Two more things that we still need to do for the new pages:
- we need to add them to the header
- we need to give them "accent" colors

We can add things to the header by editing the variable `wdw_pages` in `_config.yml`, and we can give them accent colors by populating colors into the `wdw-colors` variable in `assets/css/main.scss`.

If you want to take care of those, feel free, otherwise I can handle it.

**Comment by nicoleang** [comment]
I wasn't sure what colors the headers should be (whether they should be different colors or the same). I used darker colors (dark purple & orange) since I noticed that a darker purple was commented out in the code from before.

**Review by jez — APPROVED** [review]

**Comment by jez** [comment]
> I wasn't sure what colors the headers should be

Yeah that's fine; it doesn't matter too much, and the colors got rearranged since we added and removed some talks. I'll probably address this along with a few other changes I have to make.

Other than that, everything looks great! Feel free to hit the "Squash and merge" button whenever.

**Pull request merged**